CONTRIBUTING.rst

```diff
@@ -179,14 +199,13 @@ but will cause spurious test failures.
 Tip: If you have problems getting everything set up, consider looking at these files:
 
 -   for Linux: ``.github/workflows/test.yml`` and ``admin/Dockerfile.qa``
--   for Windows: ``.github/workflows/test.yml`` and ``appveyor.yml``
+-   for Windows: ``.github/workflows/test.yml``
 
 On **Windows**, you will need to install a GCC toolchain
 as the tests expect a Unix-like environment.
 You can use MinGW-W64 or MinGW.
-To run the tests,
-please make sure that the ``make`` and ``cmake`` from your MinGW distribution
-are in the system ``PATH``.
+To run the tests, please make sure that the ``make`` and ``cmake`` from
```

Comment:
do we also need cmake?

Reply:
Yes, there are also some tests for CMake.

Reply:
ninja is missing as a requirement...